orhan.js

```diff
@@ -0,0 +1,102 @@
+//1. Part of Homework
+
+console.log('Merhaba, Dunya!\nHello, World!\nHallo, Wereld!');
+
+//2. Part of Homework
+
+console.log('I\'m awesome');
+//this also works console.log("I'm awesome");
+
+//3. Part of Homework
+
+let x;
+console.log('the value of my variable x will be: undefined');
+console.log(x);
+x = 10;
+console.log('the value of my variable x will be: 10');
+console.log(x);
+
+//4. Part of Homework
+
+let y = 'hello';
+console.log('the value of my variable y will be: hello');
+console.log(y);
+y = 'hi';
+console.log('the value of my variable y will be: hi');
+console.log(y);
+
+//5. Part of Homework
+
+let z = 7.25;
+console.log('the value of z is: ' + z);
+let a = Math.round(z);
+console.log('the value of a is: ' + a);
+let highestValue = a < z ? z : a;
+console.log('the highest value is: ' + highestValue);
+
+//6. Part of Homework
+
+let favoriteAnimals = [];
+console.log('the value of the array will be: []');
+console.log(favoriteAnimals);
+favoriteAnimals = ['bird', 'cat'];
+console.log(favoriteAnimals);
+favoriteAnimals[2] = 'baby pig';
+console.log(favoriteAnimals);
+
+//7. Part of Homework
+
+let testString = 'this is a test';
+console.log(testString);
+let stringLength = testString.length;
+console.log(stringLength);
+
+//8. Part of Homework
+
+let myNumber = 28;
+let myString = 'sea'
+let myBoolean = true;
+let myArray = [1, 2, 3, 4, 5];
```

Comment:
Can use `const` here again instead of `let` as we are not re-assigning any of these variables, right? 🙂